modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-cli_{context}"]
+= EUS-to-EUS update using the CLI
+
+.Prerequisites
+
+* Update OpenShift CLI `oc` to the target version before each update.
+
+.Procedure
+
+. Upgrade any OLM Operators to versions that are compatible with both versions you are updating to.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
+To view the status of all machine config pools, run the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+Output is trimmed for clarity:
++
+[source,terminal]
+----
+NAME     CONFIG                                         	UPDATED   UPDATING
+master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
+worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
+----
+
+. Change to the `eus-4.12` channel. Run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade channel eus-4.12
+----
++
+[NOTE]
+====
+The `oc adm upgrade channel` command is only present in 4.9 or later.
+
+If you receive an error message indicating that `eus-4.12` is not one of the
+available channels, this indicates that Red Hat is still rolling out 4.10 to 4.12 EUS upgrades.
+This rollout process generally takes 45-90 days starting at the GA date.
+====
++
+
+. Pause the machine config pools that you want to skip reboots on. Run the following commands:
++
+[NOTE]
+====
+You cannot pause the master pool.
+====
++
+[source,terminal]
+----
+$ oc patch mcp/worker --type merge --patch '{"spec":{"paused":true}}'
+----
+
+. Update to version 4.11. Run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-latest
+----
++
+.Example output
++
+[source,terminal]
+----
+Updating to latest version 4.11.18
+----
+
+. Ensure that the 4.11 updates completed by reviewing the cluster version. Run the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME  	  VERSION  AVAILABLE  PROGRESSING   SINCE   STATUS
+version   4.11.18   True       False         6m29s   Cluster version is 4.11.18
+----
+
+. If necessary, upgrade OLM Operators by using the Administrator perspective on the web console.
+
+. Update to version 4.12. Run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-latest
+----
+
+. Ensure that the 4.12 update completed by retrieving the cluster version. Run the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME  	  VERSION  AVAILABLE  PROGRESSING   SINCE   STATUS
+version   4.12.1   True       False         6m29s   Cluster version is 4.12.1
+----
+
+. Unpause all previously paused machine config pools. Run the following command:
```

Comment:
we may want to rephrase into something like `update your worker nodes to 4.12 by unpausing all previously paused machine config pools`, which would also imply that the admin is expected to wait for a while before proceeding to the next step of verifying the workers

Reply:
can do!